.github/workflows/build-sample-app-for-sdk-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Build sample app for SDK release
+on:
+  workflow_dispatch:
+jobs:
+  build-sample-app:
+    runs-on: ubuntu-latest
+    name: Build sample app
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: MBL-553-build-sample-app-for-sdk-release
+          fetch-depth: 0 # Workaround for bug https://github.com/actions/checkout/issues/1471
+
+      - name: Get latest SDK version
+        id: latest-sdk-version-step
+        run: |
+          latest_tag=$(git describe --tags --abbrev=0)
+          echo "LATEST_TAG=$latest_tag" >> "$GITHUB_OUTPUT"
+
+      - uses: ./.github/actions/setup-android
+
+      - name: Install Ruby used by Fastlane
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
+          bundler-cache: true
+
+      - name: Setup local.properties file for sample app
+        env:
+          SDK_VERSION: ${{ steps.latest-sdk-version-step.outputs.LATEST_TAG }}
+        run: |
+          touch "samples/local.properties"
+          echo "cdpApiKey=${{ secrets.CUSTOMERIO_JAVA_WORKSPACE_CDP_API_KEY }}" >> "samples/local.properties"
+          echo "siteId=${{ secrets.CUSTOMERIO_JAVA_WORKSPACE_SITE_ID }}" >> "samples/local.properties"
+          echo "sdkVersion=${{ env.SDK_VERSION }}" >> "samples/local.properties"
```

Comment:
The sample app build scripts already check for `sdkVersion` [here](https://github.com/customerio/customerio-android/blob/main/samples/sample-app.gradle#L67) and this will cause the public release to be used instead of local project